src/transport/SessionManager.cpp

```diff
@@ -117,6 +117,11 @@ void SessionManager::Shutdown()
     mCB           = nullptr;
 }
 
+void SessionManager::SyncRemovalFabricIndex(FabricIndex fabricIndex)
```

Comment:
This should probably be called `FabricRemoved`.

And it should probably document why it's not calling `ExpireAllPairingsForFabric` (because we might still be trying to use the CASE session for the fabric to send a response, right?).